### PR TITLE
dbeaver/pro#2283 fix incorrect prompt conditions & save settings

### PR DESCRIPTION
**Pull request by Destrolaric**
Closes dbeaver/pro#2283
Fix incorrect prompt conditions

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]
Please test against OpenAI and Gemini models.

File: plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/metadata/MetadataProcessor.java [hunk 1]
```diff
@@ -122,9 +122,10 @@ public DAICompletionMessage createMetadataMessage(
                 "Your query must start with \"SELECT\" and be enclosed with triple backslash on new lines. " +
                 "Talk naturally, as if you were talking to a human.");
         } else if (isChatAPI) {
-            sb.append("Perform SQL completion. Start response with SELECT keyword. Start every line with \"--\".");
+            sb.append("Perform SQL completion. Start response with SELECT keyword.\n" +
+                " Avoid using Markdown. Start every comment line with \"--\".");
         } else {
-            sb.append("Perform SQL completion.");
+            sb.append("Perform SQL completion. Avoid using Markdown");
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/metadata/MetadataProcessor.java:128]: Probably lines 124-129 could be just
```java
        } else {
            sb.append("Perform SQL completion. Start your response with \"SELECT\".");
        }
```
Like it was before

**Review by ShadelessFox — CHANGES_REQUESTED** [review]

**Comment by Matvey16** [comment]
Verified

File: plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/metadata/MetadataProcessor.java (hunk 1 above)
Comment (Destrolaric) [reply plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/metadata/MetadataProcessor.java:128]: No, unfortunately Gemini tends to use markdown if not discouraged

**Pull request merged**